src/deconvolution.jl

```diff
@@ -38,6 +38,10 @@ regularizers and mappings.
                      and extended with a mean value (if border regions are used).
 - `debug_f=nothing`: A debug function which must take a single argument, the current reconstruction.
 
+!!! note
+    If you want to provide your PSF model, ensure that it is not centered. You may need to use `fftshift` for a PSF
```

Comment:
maybe put: "ensure that it is centered around the first entry of the array (`psf[1]`).
And: "You may need to use `ifftshift`".

Shifting from center `size(psf, 1) \div 2 + 1` to `1` requires a `ifftshift